pallets/chain-space/src/tests.rs

```diff
@@ -4,6 +4,7 @@ use codec::Encode;
 use cord_utilities::mock::{mock_origin::DoubleOrigin, SubjectId};
 use frame_support::{assert_err, assert_ok, error::BadOrigin};
 use frame_system::RawOrigin;
+use identifier::IdentifierError;
```

Comment:
This will be part of identifier tests.
Move it to primitives/identifier module.